src/xcode/ENA/ENA/Source/Scenes/Vaccination/VaccinationQRCodeScannerViewController.swift

```diff
@@ -0,0 +1,250 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import AVFoundation
+
+class VaccinationQRCodeScannerViewController: UIViewController {
+	// MARK: - Init
+
+	init(
+		didScanCheckin: @escaping (String) -> Void,
+		dismiss: @escaping () -> Void
+	) {
+		self.dismiss = dismiss
+		
+		super.init(nibName: nil, bundle: nil)
+		
+		self.viewModel = VaccinationQRCodeScannerViewModel(
+			onSuccess: { [weak self] qrCodeString in
+				AudioServicesPlaySystemSound(SystemSoundID(kSystemSoundID_Vibrate))
+				self?.viewModel?.deactivateScanning()
+			//	didScanCheckin(qrCodeString)
```

Comment:
Is this needed?

Reply:
No i just left it to indicate we need to call the closure here, i will remove it